src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  errorMessage: string,
+  setErrorMessage: (str: string) => void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={cn(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        {
+          hidden: !errorMessage,
+        },
+      )}
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => setErrorMessage('')}
```

Comment:
```suggestion
        onClick={resetMessage}
```
Consider moving this logic from jsx